app/models/health/patient.rb

```diff
@@ -688,6 +688,29 @@ def current_housing_status
       # end
     end
 
+    def last_outreach_enrollment_date(user)
```

Comment:
Let's make this `client.service_history_enrollments.visible_in_window_to(user).entry.ongoing.so.max(:first_date_in_program)`